pom.xml

```diff
@@ -137,11 +159,16 @@
             <groupId>org.springframework.kafka</groupId>
             <artifactId>spring-kafka-test</artifactId>
             <scope>test</scope>
+            <version>3.0.13</version>
```

Comment:
Please define version in properties section.

Reply:
good spot, thanks